src/librbd/internal.cc

```diff
@@ -3762,7 +3762,8 @@ int mirror_image_disable_internal(ImageCtx *ictx, bool force,
   };
 
   void readahead(ImageCtx *ictx,
-                 const vector<pair<uint64_t,uint64_t> >& image_extents)
+                 const vector<pair<uint64_t,uint64_t> >& image_extents,
+                 ZTracer::Trace *trace)
```

Comment:
I think it's unnecessary for readahead since it's a system-generated IO request instead of a traced, user-generated IO request.